scopes/react/react/react.docs.md

```diff
@@ -0,0 +1,422 @@
+---
+description: A Bit development environment for React Components
+labels: ['react', 'env', 'typescript', 'extension']
+---
+
+The React environment spares you the overhead of setting up your own development environment for React components in a Bit workspace.
+Instead, this pre-configured environment can be added to your workspace to let you instantly start working on your components. 
+
+As with any other Bit environment, the React environment is easily extendable so that you can customize it to your own needs and share it with your team to speed up development and standardize it across various decoupled workspaces.
+The React environment uses various services, provided by other Bit extensions, to handle the life events of React components, managed in a Bit workspace.
+
+#### Features
+
+- **Blazing fast environment setup**: Setting up the React environment requires nothing more than configuring a workspace to use this Bit extension.
+  Get started in seconds and focus on the thing that matters most - delivering great features.
+
+- **Less to learn**: Using the React environment means you don't have to get into all the details of your build tooling.
+  That's a especially important when onboarding a new developer to your team.
+
+- **Customizable and extensible**: React can be extended to add or override configurations.
+  Quickly add your own modifications to get a React environment "flavour" that best suits your needs.
+  Export your React environment extension to a remote scope to have it available to all your team.
+
+- **Standardized development**: Use the React environment to maintain consistency in development across multiple decoupled Bit workspaces.
+
+- **Easy to maintain**: Get React's latest updates with just a simple `bit import` command. Roll-back as easily, if needed.
+
+## Quickstart & configuration
+
+> To use the React environment, set it in the `workspace.jsonc` configuration file. React can only be configured using the 'variants' config API.
+
+### Use React as the default environment
+
+Apply the React environment on all components in the workspace, using the wildcard character `*`.
+
+```json
+{
+  "teambit.workspace/variants": {
+    "*": {
+      "teambit.react/react": {}
+    }
+  }
+}
+```
+
+### Use React on a specific group of components
+
+Apply the React environment on a limited set of components. For example, all components inside the `components/react-ui` directory.
+
+```json
+{
+    "teambit.workspace/variants": {
+    "*": {
```

Comment:
wrong example - change to `components/react-ui`

Reply:
Done (sorry for that. too many edits)